client/internal/routemanager/vars/vars.go

```diff
@@ -13,4 +13,6 @@ var (
 
 	Defaultv4 = netip.PrefixFrom(netip.IPv4Unspecified(), 0)
 	Defaultv6 = netip.PrefixFrom(netip.IPv6Unspecified(), 0)
+
+	ExitNodeCIDR = "0.0.0.0/0"
```

Comment:
should this be a const?

Reply:
it's getting hard to keep track hardcoded values